dashboard/src/app/component/partial/SilencesList/SilencesListItem.js

```diff
@@ -115,20 +116,18 @@ class SilencesListItem extends React.Component {
                 paddingTop: 8, // one spacing unit
               }}
             >
-              <Maybe value={silence.creator}>
```

Comment:
I think this may be a regression that was introduced in the rebase. Was introduced in #2817.

Reply:
good catch! This is one of two conflicts I had to resolve